src/Wpf.Ui/Resources/Theme/Dark.xaml

```diff
@@ -17,10 +17,10 @@
     -->
 
     <Color x:Key="ApplicationBackgroundColor">#FF202020</Color>
-    <SolidColorBrush x:Key="ApplicationBackgroundBrush" Color="{StaticResource ApplicationBackgroundColor}" />
+    <SolidColorBrush x:Key="ApplicationBackgroundBrush" Color="{DynamicResource ApplicationBackgroundColor}" />
```

Comment:
Do we really need a dynamic resource if the declaration is before it?

Reply:
And if so, it may be worth rebuilding the theme files to contain only Colors and Brushes in a separate, common one.